meerk40t/balormk/gui/balorcontroller.py

```diff
@@ -104,7 +104,7 @@ def on_usb_update(self, origin=None, status=None):
         if status is None:
             status = "Unknown"
         try:
-            connected = self.service.connected
+            connected = self.service.driver.connected
         except AttributeError:
```

Comment:
The except attribute error here would have let this error be detected otherwise. Though I guess a different `device` might invoke something different for that, and crash there. Seems like it should maybe not check for that error but rather correctly check if the type of device is actually a balor device somehow.